database/integration_test.go

```diff
@@ -1278,19 +1278,33 @@ func testSchedules(t *testing.T, db Interface, resources *Resources) {
 		methods[element.Method(i).Name] = false
 	}
 
-	ctx := context.TODO()
+	// create owners
+	for _, user := range resources.Users {
+		_, err := db.CreateUser(context.TODO(), user)
+		if err != nil {
+			t.Errorf("unable to create user %d: %v", user.GetID(), err)
+		}
+	}
+
+	// create the repos
+	for _, repo := range resources.Repos {
+		_, err := db.CreateRepo(context.TODO(), repo)
+		if err != nil {
+			t.Errorf("unable to create repo %d: %v", repo.GetID(), err)
+		}
+	}
 
 	// create the schedules
 	for _, schedule := range resources.Schedules {
-		_, err := db.CreateSchedule(ctx, schedule)
+		_, err := db.CreateSchedule(context.TODO(), schedule)
```

Comment:
what is the reason for inlining the context?

Reply:
I noticed that the majority of the other test functions had the context inline and wanted to be consistent.